src/util/Notation/Notation.ts

```diff
@@ -0,0 +1,109 @@
+import {
```

Comment:
Is this our own code or borrowed from some Internet resources ?

Reply:
my code moved from type transform file to utils, because only types should be there.